kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/MavenRepositoryConfiguration.java

```diff
@@ -155,11 +158,15 @@ private boolean isProfileActive( Profile profile ) {
                                                      .addPassword( server.getPassword() )
                                                      .build() );
         }
-        if (settings.getActiveProxy() != null) {
-            remoteBuilder.setProxy(getActiveAetherProxyFromSettings(settings));
+
+        if ( activeProxy != null ) {
+            if ( null == activeProxy.getNonProxyHosts() ) {
```

Comment:
Sure thing, will update it. but, as we discussed on chat, it confuses me, I am using the same code style used on Kogito, if I do apply it on this java file, a lot of formatting changes will be done, and I don't want to make changes that do not relate with the issue in specific. seems that this class has more than 1 code style in there.

for the future, it might worth apply the same pattern to all classes.